test/controller.test.ts

```diff
@@ -1502,6 +1502,398 @@ contract('Controller', ([owner, accountOwner1, accountOperator1, random]) => {
     })
   })
 
+  describe('Short otoken', () => {
+    let longOtoken: MockOtokenInstance
+    let shortOtoken: MockOtokenInstance
+
+    before(async () => {
+      const expiryTime = new BigNumber(60 * 60 * 24) // after 1 day
+
+      longOtoken = await MockOtoken.new()
+      shortOtoken = await MockOtoken.new()
+      // init otoken
+      await longOtoken.init(
+        addressBook.address,
+        weth.address,
+        usdc.address,
+        usdc.address,
+        new BigNumber(250).times(new BigNumber(10).exponentiatedBy(18)),
```

Comment:
I have a function called `createScaledNumber()` from `utils.ts`, you can use that for creating BigNumber scaled by 1e18.

Reply:
Yeah, but that function return string, I like using BigNumber everywhere, convert to string only at `assert`.